CONTRIBUTING.md

```diff
@@ -53,6 +53,37 @@ bin/cuke [<folder>...]
 ```
 
 
+## Pull Requests
+
+Each pull request (PR) should mention the [issue](https://github.com/Originate/git-town/issues)
+it implements in the PR description:
+
+```
+implements #123
+```
+
+When merging a PR, the message for the squash commit should follow the
+[formatting guidelines for commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+and
+[mention the closed issue](https://help.github.com/articles/closing-issues-via-commit-messages)
+in the commit body, so that the respective issue is automatically closed.
+
+```
+Add a foo feature to "git bar"
+
+This allows developers to optionally foo their code before
+they finally bar it into its own feature branch.
+
+Implements #123
+```
+
+Pull requests from core developers with write access should be submitted using
+`git ship`, i.e. squash-merged.
+Pull requests from external developers should be squashed into a single commit
+in their feature branch, and then accepted by a core developer
```

Comment:
Is the part after the comma necessary? It seems to be explaining a very basic thing, and if someone is contributing to Git Town, they should already know how that works?